__tests__/workers/opportunity/storeCandidateApplicationScore.ts

```diff
@@ -0,0 +1,245 @@
+import { expectSuccessfulTypedBackground, saveFixtures } from '../../helpers';
+import { storeCandidateApplicationScore as worker } from '../../../src/workers/opportunity/storeCandidateApplicationScore';
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../../../src/db';
+import { typedWorkers } from '../../../src/workers';
+import { User, Organization } from '../../../src/entity';
+import { OpportunityMatch } from '../../../src/entity/OpportunityMatch';
+import { Opportunity } from '../../../src/entity/opportunities/Opportunity';
+import { usersFixture } from '../../fixture';
+import {
+  opportunitiesFixture,
+  organizationsFixture,
+} from '../../fixture/opportunity';
+import { ApplicationScored } from '@dailydotdev/schema';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+describe('storeCandidateApplicationScore worker', () => {
+  beforeEach(async () => {
+    jest.resetAllMocks();
+    await saveFixtures(con, Organization, organizationsFixture);
+    await saveFixtures(con, User, usersFixture);
+    await saveFixtures(con, Opportunity, opportunitiesFixture);
+  });
+
+  it('should be registered', () => {
+    const registeredWorker = typedWorkers.find(
+      (item) => item.subscription === worker.subscription,
+    );
+
+    expect(registeredWorker).toBeDefined();
+  });
```

Comment:
We technically don't need these tests anymore, since we have a global test for workers.
https://github.com/dailydotdev/daily-api/blob/6ec435845c3a30017ca2f06fe09c390f84bb3ed5/__tests__/workers/workers.ts